numpy/fft/tests/test_pocketfft.py

```diff
@@ -200,6 +200,26 @@ def test_axes(self, op):
             tr_op = np.transpose(op(x, axes=a), a)
             assert_allclose(op_tr, tr_op, atol=1e-6)
 
+    @pytest.mark.parametrize("op", [np.fft.fftn, np.fft.ifftn])
```

Comment:
Can probably test `fft2` and `ifft2` here too. Not sure about the real transforms since the shape doesn't map the same.